internal/provider/resource_string.go

```diff
@@ -21,9 +24,21 @@ func resourceString() *schema.Resource {
 		// [SDK documentation](https://github.com/hashicorp/terraform-plugin-sdk/blob/main/helper/schema/resource.go#L91).
 		MigrateState:  resourceRandomStringMigrateState,
 		SchemaVersion: 1,
-		Schema:        stringSchemaV1(false),
+		Schema:        stringSchemaV1(),
 		Importer: &schema.ResourceImporter{
-			StateContext: importStringFunc(false),
+			StateContext: importStringFunc(),
 		},
 	}
 }
+
+func importStringFunc() schema.StateContextFunc {
```

Comment:
Same as above: having this function returning another function seem not necessary.

But I might be missing something.

Reply:
Have removed.